tests/e2e/utils/KubernetesCommandLineToolsExecutor.ts

```diff
@@ -0,0 +1,131 @@
+import { echo, exec, ShellString } from 'shelljs';
+import { KubernetesCommandLineTool, TestConstants } from '../constants/TestConstants';
+import { Logger } from './Logger';
+import { ShellExecutor } from './ShellExecutor';
+
+export class KubernetesCommandLineToolsExecutor extends ShellExecutor {
+    private static container: string;
+    private static pod: string;
+    private readonly namespace: string;
+    private readonly workspaceName: string | undefined;
+    private readonly KUBERNETES_COMMAND_LINE_TOOL: string = TestConstants.TS_API_TEST_KUBERNETES_COMMAND_LINE_TOOL;
+
+    constructor(_workspaceName?: string, _namespace?: string) {
+        super();
+        this.workspaceName = _workspaceName;
+        this.namespace = this.setNamespace(_namespace);
+    }
+
+    get getWorkspaceName(): string {
+        return <string>this.workspaceName;
+    }
+
+    get getNamespace(): string {
+        return this.namespace;
+    }
+
+    loginToOcp(): void {
+        if (this.KUBERNETES_COMMAND_LINE_TOOL === KubernetesCommandLineTool.OC) {
+            Logger.debug(`${this.getLoggingName(this.loginToOcp.name)}: Login to the \'OC\' client`);
+            const url: string = this.getServerUrl();
+            Logger.debug(url, TestConstants.TS_SELENIUM_OCP_USERNAME);
+            exec(`sleep 5
+            oc login --server=${url} -u=${TestConstants.TS_SELENIUM_OCP_USERNAME} -p=${TestConstants.TS_SELENIUM_OCP_PASSWORD} --insecure-skip-tls-verify`);
+        } else {
+            Logger.debug(`${this.getLoggingName(this.loginToOcp.name)}: Kubernetes command line tool not "openshift client"`);
```

Comment:
Error message doesn't explain the root problem, that `kubectl doesn't support login command`.

Reply:
okay

Reply:
thank you